config-provisioning/src/main/java/com/yahoo/config/provision/ParentHostNotReadyException.java

```diff
@@ -0,0 +1,18 @@
+// Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.config.provision;
+
+/**
+ *
+ * Exception thrown when trying to activate a node that runs on a host that is not
+ * yet ready to run the node.
+ * 
+ * @author freva
+ *
+ */
+public class ParentHostNotReadyException extends RuntimeException {
```

Comment:
The name of this can be somewhat confusing since ready here does not refer to the node state in `node-repository`, while for `NodeRepositoryProvisioner`, this is only thrown when host is not in `active` state...

But because this exception is defined outside `node-repository` and is more general, I think this is fine, though I am open to better suggestions.

Reply:
How about `ParentHostUnavailableException`?